docs/sidebars.js

```diff
@@ -1,16 +1,16 @@
 module.exports = {
-  someSidebar: [
+  default: [
```

Comment:
at the bottom of the sidebar I see "TODO: ADD ITEMS HERE", it seems to be in the theme itself:
<img width="280" alt="Screenshot 2020-09-08 at 14 27 12" src="https://user-images.githubusercontent.com/1931615/92476565-761f8680-f1df-11ea-858c-a0a8794ef690.png">

Reply:
Yes, there are some points which come after the menu, including the link for the API docs. Current design looks like this. These things will be solved before launch or removed

![image](https://user-images.githubusercontent.com/1242864/92488332-b1758180-f1ee-11ea-8418-895e76db2206.png)

Reply:
can we remove the TODO then? Otherwise it will land on the docs for the alpha, once merged into master. We cannot merge it like that :/